theme/common.less

```diff
@@ -1636,7 +1636,7 @@ p.ui.font.small {
     margin: 0 auto;
 }
 
-.download-dialog-image img {
+.download-dialog-image img:not(.large) {
```

Comment:
Is `.download-dialog-image img` used anywhere else? If not, should we just remove this?

Reply:
The other images in the browser download + webusb pairing flow all hit this